### ISSUE TITLE
Implement ModelProfile for lemp10 and darp7

### ISSUE THREAD

**Issue by jackpot51** [pop-os/system76-power#236]
Currently only the galp5 and lemp9 are supported. These profiles can significantly change how the Battery and High Performance mode work, making each one more effective.

**Comment by curiousercreative** [comment]
I imagine we can copy the galp5 profile, same line of CPUs: https://github.com/pop-os/system76-power/pull/212/files#diff-399a4413e17da0d7eac860747b9996236cd58d790bc7835ce26f48fbe0af71a2R264-R280

**Comment by jackpot51** [comment]
@curiousercreative not quite, the galp5 has different cooling. It would be closer to the lemp9 profile.
